openarena-q3e

```diff
@@ -1,29 +1,66 @@
 #!/bin/sh
-OA_CVARS="+set fs_basegame baseoa +set protocol 71"
-DISABLED_CVARS="+set com_protocol 72 +set cl_voip 1"
 
+###
+CONFIG_FILE=".var/app/io.github.ec_.Quake3e.OpenArena/config/cfg"
```

Comment:
One more thing. This is also incorrect. We should also use a standard env variable instead here.

Reply:
`$XDG_CONFIG_HOME` if I am correct (see [this document](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) for more info). Also, if possible, please use a different name than "cfg". If I remember correctly, this file is used only to check if the OpenArena Flatpak is being run for the first time, so calling it "cfg" does not make much sense.

Reply:
Ah, I take back what I said about the config name. I see that users can also set some variables there.